Directory.Build.props

```diff
@@ -1,8 +1,10 @@
 <Project>
-    <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-        <NoWarn>CS1998</NoWarn>
-    </PropertyGroup>
-    <PropertyGroup>
-        <LangVersion>latest</LangVersion>
-    </PropertyGroup>
-</Project>
\ No newline at end of file
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>CS1998</NoWarn>
+  </PropertyGroup>
+  <PropertyGroup>
+    <LangVersion>latest</LangVersion>
+    <EnableNETAnalyzers>true</EnableNETAnalyzers>
+    <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
```

Comment:
Check for behaviour before merging. If there are too many issues with this, leave it disabled

Reply:
0 errors and 0 warnings in my VS